guides/process/scheduling/setting_up_projects.md

```diff
@@ -0,0 +1,127 @@
+# Setting up Projects
```

Comment:
Should we add this is the responsibility of the Ops team to set up projects or needs to be agreed by Ops before setting up?